assets/components/page/page.jsx

```diff
@@ -12,6 +13,7 @@ type PropTypes = {
   header: Node,
   footer: Node,
   children: Node,
+  classModifiers: Array<?string>
```

Comment:
How come `Array<?string>` and not `Array<string>`?

Reply:
It is defined as such in `classNameWithModifiers`, if I use the latter Flow completely loses it 😒

Reply:
`classNameWithModifiers` allows you to pass `null` and `undefined` (and empty string) in the list, and filters them out before building the `className`. I believe it did this for legacy reasons, to allow for situations where we were passing these values.

That said, I agree that a better type would be `Array<string>`; that would make it a single-purpose function, and it would be up to the call site to filter out `null` and `undefined`.